src/libdredd/src/mutation_replace_binary_operator.cc

```diff
@@ -232,6 +267,32 @@ std::string MutationReplaceBinaryOperator::GenerateMutatorFunction(
       << clang::BinaryOperator::getOpcodeStr(binary_operator_.getOpcode()).str()
       << " " << arg2_evaluated << ";\n";
 
+  const clang::BuiltinType* exprType =
+      binary_operator_.getType()->getAs<clang::BuiltinType>();
+
+  GenerateBinaryOperatorReplacement(operators, arg1_evaluated, arg2_evaluated,
+                                    new_function, mutant_offset);
+  GenerateArgumentReplacement(arg1_evaluated, arg2_evaluated, new_function,
+                              mutant_offset);
+  MutationReplaceExpr::GenerateUnaryOperatorInsertion(
+      GetExpr(ast_context), binary_operator_, *exprType, new_function,
+      mutant_offset);
+  MutationReplaceExpr::GenerateConstantReplacement(
+      binary_operator_, *exprType, ast_context, new_function, mutant_offset);
+
+  new_function << "  return " << GetExpr(ast_context) << ";\n";
+  new_function << "}\n\n";
+
+  // The function captures |mutant_offset| different mutations, so bump up
+  // the mutation id accordingly.
+  mutation_id += mutant_offset;
+
+  return new_function.str();
+}
+void MutationReplaceBinaryOperator::GenerateBinaryOperatorReplacement(
```

Comment:
Leave a space between the closing brace of the previous function and this opening brace.

Nit: I suggest placing this function right after GenerateArgumentReplacement, since they are analogous.

Reply:
Done.